.github/workflows/qemu-kvm-integration-tests.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         if: steps.check_platform.outputs.supported
         run: >-
           tox -e ${{ matrix.scenario.env }} -- --image-name ${{ matrix.scenario.image }} --make-batch
-          --log-level=debug --skip-tags tests::infiniband,tests::nvme,tests::scsi --
+          --log-level=debug --skip-tags tests::infiniband --
```

Comment:
This needs to be brought back to avoid the ENOSPC/too many tests. Sorry, forgot about this earlier.

Reply:
Sent https://github.com/linux-system-roles/.github/pull/101 to make this official, and I reverted this bit in this branch.